### ISSUE TITLE
Black flickering on window resizing

### ISSUE THREAD

**Issue by georg-eckert-zeiss** [MarcStan/monogame-framework-wpfinterop#21]
Hi there,

first of all: thanks for your great work : )

I have encountered an issue with resizing. I had the same in MonoGame.WpfCore. And it might be related to an older issue (see it on the older [BitBucket issue](https://gitlab.com/MarcStan/monogame-framework-wpfinterop/-/issues/12)). Any chance to get that fixed?

Cheers,
Georg

# Steps to reproduce

1. Launch the test WPF-App
2. Resize the window (smaller, bigger, faster slower)
3. Black flickering

**Comment by BradleyUffner** [comment]
Isn't this same issue present in off the shelf version of MonoGame (and even XNA)?  I remember digging through code trying to find a way to resolve it years ago.

**Comment by georg-eckert-zeiss** [comment]
I guess you are right. I forked this repo and upgrade it to MonoGame 3.8.0.1375-develop (see [here](https://github.com/georg-eckert-zeiss/monogame-framework-wpfinterop)) and the flickering is gone for good :)

I am not sure if it is fit for a pull request.

**Comment by georg-eckert-zeiss** [comment]
Unfortunately there is a bug in 3.8.0.1375-develop which prevents the included FBX from being imported - so that part is broken for now. A fix is already merged upstream but not in the NuGet yet.

**Comment by MarcStan** [comment]
Hey thanks for the investigation/upgrade!

I'll be sure to update once 3.8 is officially released (keeping this open until then)!

**Comment by MarcStan** [comment]
The latest version (1.8) should fix this.

**Comment by georg-eckert-zeiss** [comment]
Great news : ) Thank you for your work.

**Comment by georg-eckert-zeiss** [comment]
Yeah. Same here. : ( I looked into it. Definitely not an easy fix.

**Comment by laughsuggestion** [comment]
> Yeah. Same here. : ( I looked into it. Definitely not an easy fix.

Understood, can we get this reopened?
